src/main.js

```diff
@@ -59,6 +61,30 @@ let load = base.loader({
     },
   },
 
+  WorkerState: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let WorkerState = workerState.setup({
+        account: cfg.azure.account,
+        table: cfg.app.workerStateTableName,
+        credentials: cfg.taskcluster.credentials,
+      });
+      return WorkerState;
+    },
+  },
```

Comment:
yes, it is a merge error! probably it was from the commit last week and I didn't see this part was removed. Now I see the importance of sending small patches as soon as I can :)